<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPlayer</class>
 <widget class="QWidget" name="CPlayer">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>27</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="m_play">
     <property name="toolTip">
      <string>Start/Stop</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="wtools.qrc">
       <normaloff>:/icons/player_play.png</normaloff>:/icons/player_play.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_backward">
     <property name="toolTip">
      <string>Backward one frame</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="wtools.qrc">
       <normaloff>:/icons/player_backward.png</normaloff>:/icons/player_backward.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CSlider" name="m_frameNumber">
     <property name="toolTip">
      <string>Frame number</string>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_forward">
     <property name="toolTip">
      <string>Forward one frame</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="wtools.qrc">
       <normaloff>:/icons/player_forward.png</normaloff>:/icons/player_forward.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_averageMax">
     <property name="toolTip">
      <string>Date on which the average is highest</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="wtools.qrc">
       <normaloff>:/icons/max.png</normaloff>:/icons/max.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_amm">
     <property name="toolTip">
      <string>Date on which the average or the maximum is the highest or the lowest</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="wtools.qrc">
       <normaloff>:/icons/amm.png</normaloff>:/icons/amm.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="m_frameRate">
     <property name="toolTip">
      <string>Frame rate (images/s)</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="minimumContentsLength">
      <number>8</number>
     </property>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1/2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1/4</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CSlider</class>
   <extends>QSlider</extends>
   <header>../wtools/slider.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="wtools.qrc"/>
 </resources>
 <connections/>
</ui>
